src/constants.js

```diff
@@ -39,3 +39,9 @@ export const validationErrorTypes = {
 	fieldIsRequired: "fieldIsRequired",
 	fieldHasWrongValue: "fieldHasWrongValue",
 };
+
+export const mode = {
```

Comment:
mode of what? A more explicit name would be appropriate.

Reply:
done